bot.ml

```diff
@@ -87,6 +87,10 @@ let extract_commit json =
         sha )
     |> to_string
 
+let extract_commit_message json =
+  let open Yojson.Basic.Util in
+  json |> member "commit" |> member "message" |> to_string
```

Comment:
As highlighted by how the `extract_commit` function above works, this commit message is sometimes of the form "Bot merge ... into ...", which would be pretty uninformative. Therefore, it would be best to request the commit message of the actual PR head in the `get_pull_request_refs` request.

Furthermore, as suggested by @SkySkimmer, it would be best to limit to printing the first line of the commit message, as commit messages can be arbitrarily long, and we should avoid @coqbot posting messages that are too long.